packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -245,13 +246,18 @@ class CpuProfileData {
       return null;
     }
 
+    final updatedMetaData = originalData.profileMetaData.copyWith(
+      sampleCount: filteredCpuSamples.length,
```

Comment:
do we also need to update the time here like we did for the filter by user tag case?

Reply:
Done.